packages/documentation-v7/src/stories/components/collapsible/collapsible.demo.stories.ts

```diff
@@ -0,0 +1,143 @@
+import { spread } from '@open-wc/lit-helpers';
+import { useArgs } from '@storybook/preview-api';
+import { Meta, StoryContext, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { BADGE } from '../../../../.storybook/constants';
+import { definedProperties } from '../../../utils';
+
+const meta: Meta<HTMLPostCollapsibleElement> = {
+  title: 'Hidden/demos/components/Collapsible',
+  component: 'post-collapsible',
+  parameters: {
+    controls: {
+      exclude: ['Content'],
+    },
+    badges: [BADGE.BETA, BADGE.NEEDS_REVISION],
+  },
+  args: {
+    innerHTML: `<span slot="header">Titulum</span><p>Contentus momentus vero siteos et accusam iretea et justo.</p>`,
+  },
+  argTypes: {
+    innerHTML: {

```

Comment:
@alizedebray I'm not sure anymore about `innerHTML` naming. What about using `Slot`? I could extend the type as previously.

Reply:
To me, this control really represents the `innerHTML` of the component. In fact, running `document.querySelector('post-collapsible').innerHTML = '<span slot="header">My header.</span><span>My body.</span>';` correctly results in "My header." being shown in the header slot and "My body." being shown in the default (body) slot.

On the contrary, slots are part of the shadow DOM and are not impacted when you use the control.

But we can always configure another name in the argTypes configuration, no need to extend the type.

Reply:
Since we want to show explicitly how to integrate body and title content into the component, the name `slot` would be misleading in my view, because the example fills both the DefaultSlot and the TitleSlot with content.
If at all, the attribute would have to be called `slots`.

I'm unsure about the innerHTML code example of @alizedebray.
Wouldn't you have to select the ShadowDomRoot at least, to populate the innerHTML of the container?
If the answer is yes, it would not be correct for me to stick with the name `innerHTML`.
But please correct me if my assumption is wrong.

The original argument name of the Storybook V6 doc `content` or something similar would probably be most unambiguous one.

But please feel free to decide about this by yourself.
We can always change it, if some users say it's missleading.

Reply:
https://github.com/swisspost/design-system/assets/33580481/7d1f89f7-df5f-4f78-87e1-84e76b0e98b0